tests/conftest.py

```diff
@@ -328,6 +329,40 @@ def dataset_write_read_check(
                 assert new.data == dataset.data
 
 
+def dataset_write_read_batch_check(
```

Comment:
It seems this one can be used only for `pandas` datasettypes, so it should go to `test_pandas` or made more general (not sure how)